bitops.go

```diff
@@ -0,0 +1,339 @@
+			rootFolderCreate, err := exists(*initSubOptionRootDirectory)
+			
+			if err == nil { }
+			if rootFolderCreate {
+				rootFolderPath = *initSubOptionRootDirectory
+			}else{
+				fmt.Println("Not a valid path: ["+*initSubOptionRootDirectory+"]")
+				initCommand.PrintDefaults()
+            	os.Exit(1)
+			}
+		}
+
+		fmt.Println("Using root directory: [" + rootFolderPath + "]")
+		
+		// Generate Ops Repo
+		var projectname string
+		var fullprojectname string
+		fmt.Println(lineBreak)
+		for {
+			fmt.Println("Creating an Operations Repo!")
+			fmt.Print("What is the project name? : ")
+			fmt.Scanln(&projectname)
+
+			if projectname == "" {
+				continue
+			}
+			
+			fullprojectname = "Operations_"+projectname
+			fmt.Println("Generating Operations Repo: [" + fullprojectname + "]")
+			response := askForConfirmation(confirmResponse)
+		
+			if response {
+				fmt.Println("Creating [Operations_"+projectname+"]")
+				break
+			}
+		}
+
+		// Environment
+		environment := ConfirmEnvironment()
+		fmt.Println(lineBreak)
+
+		// Plugins
+		plugins := ConfirmTools()
+
+		// Make folders
+		var projectpath = rootFolderPath + "/" + fullprojectname + "/" + environment
+		fmt.Println("Creating ["+projectpath+"]")
+		os.MkdirAll(projectpath, os.ModePerm)
+		
+		CreateEnvironment(plugins, projectpath)
+    }
+
+	// ADD-NEW COMMAND
+	if addNewCommand.Parsed() {	
+		addNewCommandUsage := `Usage: "bitops add-new <add-new options> <add-new action> <Operations Repos Name>"
+  Actions:
+    env		- Add a new environment to an existing Operations Repo
+    tool	- Add a new tool to an existing Operations Repo Environment
+`
+		// Help command
+		if *addNewSubCommandHelp == true {
+            fmt.Println(addNewCommandUsage)
+			addNewCommand.PrintDefaults()
+            os.Exit(1)
+        }
+
+		if addNewCommand.Arg(0) == "" {
+			fmt.Println("Error: No action selected.")
+			fmt.Println(addNewCommandUsage)
+			addNewCommand.PrintDefaults()
+			os.Exit(1)
+		}
+
+		var rootFolderPath string = "."
+		// Directory SubCommand
+		if *addNewSubOptionRootDirectory != "" {
+			// Returns: Bool, Err - Checks if path to folder/file exists
+			rootFolderCreate, err := exists(*addNewSubOptionRootDirectory)
+			
+			if err == nil { }
+			if rootFolderCreate {
+				rootFolderPath = *addNewSubOptionRootDirectory
+			}else{
+				fmt.Println("Not a valid path: ["+*addNewSubOptionRootDirectory+"]")
+				addNewCommand.PrintDefaults()
+            	os.Exit(1)
+			}
+		}
+
+		fmt.Println("Using root directory: [" + rootFolderPath + "]")
+		
+		addNewSubCommand := addNewCommand.Arg(0)
+		opsRepoName := addNewCommand.Arg(1)
+
+		if opsRepoName == "" {
+			fmt.Println("Error: No Operations Repo specified")
+			fmt.Println(addNewCommandUsage)
+			addNewCommand.PrintDefaults()
+			os.Exit(1)
+		}
+
+		opsRepoPath := rootFolderPath + "/" + opsRepoName
+
+		// Ensure Operations Repo exists
+		exists, err := exists(opsRepoPath)
+		if err == nil {}
```

Comment:
Why this construct is needed `if err == nil {}`? Is it because of the issue with unused var?

You can use `_` instead: 
```suggestion
		exists, _ := exists(opsRepoPath)
```